SwrveSDK/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="Theme.InAppMessage" parent="android:Theme">
```

Comment:
Should this be a theme if its configurable? Would be nicer if all the logic for it was in one spot.